core/DataTable/Renderer/Csv.php

```diff
@@ -264,6 +266,30 @@ protected function formatValue($value)
         return $value;
     }
 
+    protected function formatFormulas($value)
+    {
+        // Excel / Libreoffice formulas may start with one of these characters
+        $formulaStartsWith = array('=', '+', '-', '@');
+
+        // remove first % sign
+        $count = 1;
+        $valueWithoutPercentSign = str_replace('%', '', $value, $count);
```

Comment:
Can you add a test to make sure it only replaces the first `%`?  For `str_replace`  the `$count` will hold how many values were replaced. It is not the number of how many values should be replaced. Just noticed this somewhere else.